### DISCUSSION TITLE
how to clip overlay area?

### DISCUSSION THREAD

**Discussion by ghost** [wkentaro/labelme#1222] [Q&A / Help]
i have a dataset from a lot of complex shape.
there are a lot of overlay area

how to clip overlay area?
![image](https://user-images.githubusercontent.com/110151053/207025759-f1a42d95-ff7c-4dbe-9814-00270d75a19a.png)
![image](https://user-images.githubusercontent.com/110151053/207025803-3aaa1cc1-2a4d-4c7d-badc-f500612fe220.png)
yellow color is road class, but they overlay. it need to clip

![image](https://user-images.githubusercontent.com/110151053/207025854-7d6bcc0d-2427-4dbe-9fc8-4c6c85f1c76f.png)

how to auto clip same class when it overlay?

**Comment by wkentaro** [comment]
That would be the job of export script after annotation.

In [this example](https://github.com/wkentaro/labelme/tree/main/examples/tutorial#visualization), the object region is clipped from the container annotation. Be careful about the order of anotations in "Polygon Labels".

**Reply by ghost** [reply]
I mean I want to only json file to clip, not png clip overlay.
Can you give a demonstration?

**Reply by wkentaro** [reply]
No that's not possible by Labelme as of now. As I believe that is the job of the software that processes that JSON file.
